pages/set/_.vue

```diff
@@ -154,7 +154,12 @@
         }
         throw apiError;
       }
+      this.getRecommendations();
+      this.$watch('set.total', function() {
+        this.getRecommendations();
+      });
     },
+    fetchOnServer: false,
```

Comment:
This doesn't make sense as it also prevents the error code from being set which only happens ` if (process.server)`.

The reason recommendations aren't being shown on SSR seems to me to be that this.$auth.loggedIn is always false while on SSR.